src/lib.rs

```diff
@@ -11,19 +11,19 @@
 //! * `GET /api/info` returns an informative [`api::InfoResponse`] object.
```

Comment:
If you are not adding `/health` here to the API, then you should not call it to check server availability as is currently done in `runner-router`. Altho I would only use `/api/info` anyway, as it is backwards compatible (at least to tell you that it is incompatible).